astroplan/core.py

```diff
@@ -2,6 +2,21 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord,
```

Comment:
I still don't understand why that line gave an error on travis-ci:
```
ImportError: No module named astropy.coordinates
```
But if you fix the other errors mentioned below and `python setup.py test` (which imports this file) works locally, it should work on travis-ci, too.